.cljfmt/indents.clj

```diff
@@ -1,2 +1,33 @@
-{metabase.api.collection/foo [[:inner 0]]
- toucan.db/insert!           [[:block 1]]}
+;; https://github.com/weavejester/cljfmt/blob/master/cljfmt/resources/cljfmt/indents/clojure.clj
+{auto-parse                                 [[:inner 0]]
+ cond->                                     [[:inner 0]]
+ check                                      [[:inner 0]]
+ deferred-tru                               [[:inner 0]]
+ defprotocol+                               [[:block 1] [:inner 1]]
+ ;ex-info                                 [[:inner 0]]
+ let-404                                    [[:block 1]]
+ mbql.u/match-one                           [[:inner 0]]
+ merge-with                                 [[:inner 0]]
+ mt/dataset                                 [[:inner 0]]
+ mt/formatted-rows                          [[:inner 0]]
+ mt/mbql-query                              [[:inner 0]]
+ mt/test-drivers                            [[:inner 0]]
+ mt/with-temp*                              [[:block 1]]
```

Comment:
We removed `mt/with-temp*`  months ago, so this file might be a little out of date 😳